docs/docs/04-standard-library/std/json.md

```diff
@@ -450,16 +450,16 @@ Convert Json element to string if possible.
 ##### `delete` <a name="delete" id="@winglang/sdk.std.MutJson.delete"></a>
 
 ```wing
-delete(key: str): bool
+delete(key: any): bool
 ```
 
 Removes the specified element from a map.
 
 ###### `key`<sup>Required</sup> <a name="key" id="@winglang/sdk.std.MutJson.delete.parameter.key"></a>
 
-- *Type:* str
+- *Type:* any
 
-The key.
+Either the key or index to delete.
```

Comment:
Im not sure we should allow deletion at an index. While it may seem like order is preserved in most JSON objects, this is not a guarantee, according to the [json specification](https://www.json.org/json-en.html) "An object is an unordered set of name/value pairs"

Reply:
It makes sense for Json Arrays, since their order is preserved but perhaps in these cases its better to just convert types